src/Task/TwigCs.php

```diff
@@ -57,6 +59,12 @@ public function run(ContextInterface $context): TaskResultInterface
         $arguments->addOptionalArgument('--ruleset=%s', $config['ruleset']);
         $arguments->addOptionalArgument('--ansi', true);
 
+        if (\is_array($config['exclude'])) {
+            foreach ($config['exclude'] as $exclude) {
```

Comment:
You can skip the foreach and use the `\GrumPHP\Collection\ProcessArgumentsCollection::addArgumentArray()` method instead

Reply:
I can have empty values in my array, so can i use addSeparatedArgumentArray() instead of addArgumentArray() ? 

Like :
```php
$arguments->addSeparatedArgumentArray('--exclude', $config['exclude']);
```

Reply:
Why could you have empty values? Don't you always want to specify locations to exclude?
You can indeed also use `addSeparatedArgumentArray` if the tool allows you.

If you want to remove empty values, you could wrap it with `array_filter($config['exclude'])` as well?

Reply:
Yes, job done @veewee :smile: